api/v1alpha1/qserv_types.go

```diff
@@ -87,7 +88,8 @@ type CzarSettings struct {
 
 // DashboardSettings defines the specification for dashoard deployment
```

Comment:
Per conversation on Slack, dashboard service and image should be completely removed for latest `lite` (could be on a follow-up ticket though)